### PR TITLE
feat: multi-platform file path support

### PR DESCRIPTION
**Pull request by davidwalschots**
Let's support Windows as well!

### CODE REVIEW

File: rollup.js [hunk 1]
```diff
@@ -46,18 +47,18 @@ module.exports = ({ name, jsBundle = 'bundle.js', cssBundle = 'bundle.css', inst
         const relativeOutputDirStart = outputDir.indexOf(OUTPUT_DIR_ROOT);
 
         if (relativeOutputDirStart === -1) {
-            this.error({ message: `outputDir must contain '${OUTPUT_DIR_ROOT}'` });
+            this.error({ message: `outputDir must contain '${OUTPUT_DIR_ROOT}', was: ${outputDir}` });
         }
 
         const relativeOutputDir = outputDir.substring(relativeOutputDirStart);
```
Comment (davidwalschots) [inline rollup.js:53]: The `relativeOutputDir` name was and is very wonky. As it doesn't contain a relative but an absolute path (i.e. it starts with a `/`). However, I decided to not touch naming and really only fix the issue I was having 😀.

File: rollup.js [hunk 2]
```diff
@@ -46,18 +47,18 @@ module.exports = ({ name, jsBundle = 'bundle.js', cssBundle = 'bundle.css', inst
...
-        const finalOutputDir = `${relativeOutputDir}/${instrumentDir}`;
+        const finalOutputDir = path.join(relativeOutputDir, instrumentDir);
 
-        const processedHtml = html(name, finalOutputDir, imports, cssCode, jsCode);
+        const processedHtml = html(name, finalOutputDir.replace(/\\/g, "/"), imports, cssCode, jsCode);
```
Comment (davidwalschots) [inline rollup.js:56]: Every line in this file relates to filesystem paths. This is the only location where it turns into a HTML compatible file path, which is always with `/`. Thus, we need to convert it here.

**Comment by Benjozork** [comment]
Did you test that this still works in WSL?

**Comment by davidwalschots** [comment]
I'm not developing on WSL. I tested that it works with `.\scripts\dev-env\run.cmd ./scripts/build.sh`.

**Pull request merged**